src/Tests/Sfa.Tl.ResultsAndCertification.Web.UnitTests/Controllers/AdminDashboardControllerTests/ChangeIndustryPlacementGet/When_Called_With_Invalid_Data.cs

```diff
@@ -0,0 +1,43 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using NSubstitute;
+using Sfa.Tl.ResultsAndCertification.Common.Helpers;
+using Sfa.Tl.ResultsAndCertification.Common.Services.Cache;
+using Sfa.Tl.ResultsAndCertification.Web.ViewModel.AdminDashboard;
+using System.Threading.Tasks;
+using Xunit;
+namespace Sfa.Tl.ResultsAndCertification.Web.UnitTests.Controllers.AdminDashboardControllerTests.ChangeIndustryPlacementGet
+{
+    public class When_Called_With_Invalid_Data: TestSetup
+    {
+        public int PathwayId { get; set; }
+        
+        protected AdminChangeIndustryPlacementViewModel Mockresult = null;
+        public IActionResult Result { get; private set; }
+
+
+        public override void Given()
+        {
+            PathwayId = 0;
+            AdminDashboardLoader.GetAdminLearnerRecordAsync<AdminChangeIndustryPlacementViewModel>(PathwayId).Returns(Mockresult);
+        }
+
+        public async override Task When()
```

Comment:
The method is declared in the parent class

Reply:
Done